RubeesEat.IntegrationTests/IntegrationTestBase.cs

```diff
@@ -18,7 +18,8 @@ public void SetUp()
     {
         TestServer = new TestServer(new WebHostBuilder()
                                     .UseStartup<Startup>()
-                                    .ConfigureTestServices(ConfigureServices));
+                                    .ConfigureTestServices(ConfigureServices)
+                                    .UseEnvironment("Test"));

```

Comment:
We should be able to revert this and do the things we need when we configure the test services.